src/components/ReleaseAnnouncement/ReleaseAnnouncement.module.css

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2024 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.content {
+  /* 320px + 20px padding left + 20px padding right */
+  max-inline-size: 320px;
+  border-radius: var(--cpd-space-3x);
+  background: var(--cpd-color-bg-action-primary-rest);
+  padding: var(--cpd-space-4x) var(--cpd-space-5x);
+
+  /* from figma */
+  box-shadow: 0 4px 24px 0 rgb(0 0 0 / 10%);
+
+  /*
+  * Grid, we want to have the following layout:
+  * ----------------------------------
+  * -  ---------------   ----------  -
+  * -  | heading     |   |        |  -
+  * -  --------------    | button |  -
+  * -  ---------------   | button |  -
+  * -  | description |   |        |  -
+  * -  ---------------   ----------  -
+  * ----------------------------------
+  */
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+  align-items: center;
+  column-gap: var(--cpd-space-6x);
+}
+
+.heading {
+  margin: 0;
+  color: var(--cpd-color-text-on-solid-primary);
+  grid-area: 1 / 1 / 2 /2;
+}
+
+.description {
+  color: var(--cpd-color-gray-500);
```

Comment:
This seems to now use color/text/on-solid/primary in the designs

Reply:
That's the case of the header but the description is on Light/color/gray/500